bin/rails

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env ruby
+
```

Comment:
you should not modify bin/rails.  this code should go into spec_helper.rb and/or features/support/env.rb.